app/views/submissions/user_actions.erb

```diff
@@ -5,6 +5,26 @@
     </button>
     <ul class="dropdown-menu" role="menu">
       <li>
+        <% if exercise.request_help? %>
+          <form accept-charset="UTF-8" action="/exercises/<%= exercise.key %>/remove_help" method="POST">
+            <button type="submit" name="reopen" class="btn btn-default">Remove request for help</button>
+          </form>
+        <% else %>
+          <form accept-charset="UTF-8" action="/exercises/<%= exercise.key %>/request_help" method="POST">
+            <button
+              type="submit"
+              name="done"
+              class="btn btn-default"
+              data-toggle="tooltip"
+              data-placement="bottom"
+              title="Help flag will be added to your exercise on activity stream.">
```

Comment:
Let's make this a bit more conversational. Maybe: _Your exercise will be marked on the activity stream as needing help._